rebac-admin-backend/v1/identities.go

```diff
@@ -78,18 +80,19 @@ func (h handler) GetIdentitiesItem(w http.ResponseWriter, req *http.Request, id
 		return
 	}
 
-	writeResponse(w, 200, identity)
+	writeResponse(w, http.StatusOK, identity)
 }
 
 // PutIdentitiesItem updates the identity identified by the provided ID.
 // (PUT /identities/{id})
 func (h handler) PutIdentitiesItem(w http.ResponseWriter, req *http.Request, id string) {
 	ctx := req.Context()
+
 	identity := new(resources.Identity)
 	defer req.Body.Close()
 
 	if err := json.NewDecoder(req.Body).Decode(identity); err != nil {
-		writeErrorResponse(w, err)
+		writeErrorResponse(w, NewValidationError("Request doesn't match the expected schema"))
```

Comment:
Just one thing, the idiomatic rule for error messages in Go is:
- No capital first letter 
- No punctuation at the end